docs/static/arcsight-module.asciidoc

```diff
@@ -26,65 +36,54 @@ contains two core data collection components for data streaming:
 
 * The _Smart Connectors (SC)_ are edge log collectors that parse and normalize
 data to CEF prior to publishing to the Logstash receiver.
-* The _Event Broker (EB)_ is the central hub for incoming data and is based on
+* The _Event Broker_ is the central hub for incoming data and is based on
 open source Apache Kafka. The Logstash ArcSight module can consume directly from
-EB topics.
+Event Broker topics.
 
 [[arcsight-getting-started-smartconnector]]
-==== Getting Started With The Smart Connector
+==== Getting Started with the Smart Connector
+
+:smart_connect_host: smart_connect_host
+:smart_connect_port: smart_connect_port
 
 To get started, you can use a basic Elastic Stack setup that reads events from
 the Smart Connector directly.
 
 image::static/images/arcsight-diagram-smart-connectors.svg[ArcSight Smart Connector architecture]
 
-[[arcsight-requirements-smartconnector]]
-===== Requirements
 
-* These instructions assume you have part of the Elastic Stack (Logstash, Elasticsearch,
-Kibana) already installed. The products you need are
-https://www.elastic.co/downloads[available to download] and easy to install. The
-Elastic Stack 5.6 or higher is required for this module.
-* The Elastic Stack is running locally with default ports exposed, namely
-Elasticsearch as "localhost:9200" and Kibana as "localhost:5601". Note that you can also run 
-Elasticsearch, Kibana and Logstash on separate hosts to consume data from ArcSight.
-* Smart Connector has been configured to publish ArcSight data (to TCP port `5000`) using the CEF syslog 
-destination.
 
-[[arcsight-instructions-smartconnector]]
-===== Instructions
 
-. {ref}/installing-xpack-es.html[Install X-Pack on Elasticsearch] and then start
-Elasticsearch.
+Smart Connector has been configured to publish ArcSight data (to TCP port `5000`) using the CEF syslog 
+destination.
 
-. {kibana-ref}/installing-xpack-kb.html[Install X-Pack on Kibana] and then start
-Kibana.
+NOTE: Logstash, Elasticsearch, and Kibana must run locally. Note that you can also run 
+Elasticsearch, Kibana and Logstash on separate hosts to consume data from ArcSight.
+
+[[arcsight-instructions-smartconnector]]
+===== Instructions for Smart Connector
 
-. {logstash-ref}/installing-xpack-log.html[Install X-Pack on Logstash], which
-includes the Logstash ArcSight module.
 . Start the Logstash ArcSight module by running the following command in the
-Logstash install directory with your respective EB host and port:
+Logstash install directory with your respective Smart Connector host and port:
 +
```

Comment:
@acchen97   @jakelandis  The instructions for Smart Connector had an EB reference. Took a chance and changed it. I can change it back if I guess incorrectly.

Reply:
lgtm